ton_client/client/src/boc/mod.rs

```diff
@@ -0,0 +1,156 @@
+/*
+* Copyright 2018-2020 TON DEV SOLUTIONS LTD.
+*
+* Licensed under the SOFTWARE EVALUATION License (the "License"); you may not use
+* this file except in compliance with the License.
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific TON DEV software governing permissions and
+* limitations under the License.
+*/
+
+use crate::dispatch::DispatchTable;
+use crate::client::ClientContext;
+use crate::error::{ApiResult};
+use ton_block::Deserializable;
+use errors::Error;
+
+mod errors;
+
+// TODO: uncomment when module will be ready
+//mod cell;
+
+#[cfg(test)]
+mod tests;
+
+#[derive(Serialize, Deserialize, Clone, TypeInfo)]
+pub struct ParamsOfParse {
+    /// BOC encoded as base64
+    pub boc: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, TypeInfo)]
+pub struct ResultOfParse {
+    /// JSON containing parsed BOC
+    pub result: serde_json::Value,
```

Comment:
Suggest do not use `result`. The `parsed` or `json` is better.
For example:
let result = client.boc.parseMessage(...);
Then:
println!("{}", result.result); // Not good
Or:
println!("{}", result.parsed); // The best
Or:
println!("{}", result.json); // Better